fuzz/tls_init/fuzz_init.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+extern "C"{
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include "rats-tls/api.h"
+#include "rats-tls/log.h"
+#include "rats-tls/claim.h"
+#include "internal/core.h"
+}
+#include <fuzzer/FuzzedDataProvider.h>
+#include <vector>
+
+#define CUSTOM_CLAIMS_SIZE 10
+using namespace std;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data,size_t size){
+    rats_tls_conf_t conf; // consume 192 bytes
+    // conf | claim_array | random char *  in claim_array
+    if(size < sizeof(rats_tls_conf_t) + 10 * sizeof(claim_t) + 50 * 10){
+        return 0;
+    }
+    memcpy(&conf, data, sizeof(conf));
+    conf.log_level  = RATS_TLS_LOG_LEVEL_DEFAULT;
+    conf.api_version = 0;
+    
+    /*fuzz log level*/
+    /*fuzz round could not be too huge, that leads to unexpected log_level*/
+    strcpy(conf.attester_type, "nullattester");
+    strcpy(conf.verifier_type, "nullverifier");
+    strcpy(conf.tls_type, "nulltls");
+    strcpy(conf.crypto_type, "nullcrypto");
+    
+    conf.cert_algo = RATS_TLS_CERT_ALGO_DEFAULT;
+    conf.flags = RATS_TLS_CONF_FLAGS_MUTUAL;
+
+    FuzzedDataProvider fuzzed_data(data + sizeof(conf), size - sizeof(conf));
+    claim_t custom_claims[CUSTOM_CLAIMS_SIZE];
+    std::vector<std::string> str_lists;
+    for(int i=0;i<CUSTOM_CLAIMS_SIZE;i++){
+        //const char * str = fuzzed_data.ConsumeBytesWithTerminator(50,'\0').data();
```

Comment:
please refer to [link](https://github.com/inclavare-containers/inclavare-containers/blob/master/CONTRIBUTING.md#format-c-codes) to format your code.